express-api/src/routes/ltsaRouter.ts

```diff
@@ -6,6 +6,6 @@ import express from 'express';
 const router = express.Router();
 
 // Endpoints for LTSA title information
-router.route('/land/title').get(activeUserCheck, catchErrors(controllers.getLTSA));
+router.route('/land/title').get(activeUserCheck, catchErrors(controllers.getToken));
```

Comment:
Same comment. Wait to assign a controller here until you have the parent function that will combine all three steps of the service.